proposal/slash-separator.md

```diff
@@ -0,0 +1,415 @@
+# Forward Slash as a Separator: Draft 1
+
+*([Issue](https://github.com/sass/sass/issues/2565))*
+
+This proposal modifies the `/` character to be used exclusively as a separator,
+and lays out a process for deprecating its existing usage as a division
+operator.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+* [Alternatives Considered](#alternatives-considered)
+  * [Different Operator Syntax](#different-operator-syntax)
+  * [First-Class `calc()`](#first-class-calc)
+  * [`math()` Syntax](#math-syntax)
+* [Existing Behavior](#existing-behavior)
+* [Syntax](#syntax)
+* [Semantics](#semantics)
+  * [Slash-Separated Lists](#slash-separated-lists)
+  * [`divide()`/`math.div()` Function](#divide-math-div-function)
+  * [`slash-list()` Function](#slash-list-function)
+  * [`rgb()` Function](#rgb-function)
+  * [`hsl()` Function](#hsl-function)
+* [Deprecation Process](#deprecation-process)
+  * [Phase 1](#phase-1)
+  * [Phase 2](#phase-2)
+  * [Phase 3](#phase-3)
+* [Timeline](#timeline)
+
+## Background
+
+> This section is non-normative.
+
+Early on in Sass's history, the decision was made to use `/` as a division
+operator, since that was (and is) by far the most common representation across
+programming languages. The `/` character was used in very few plain CSS
+properties, and for those it was an optional shorthand. So Sass defined [a set
+of heuristics][] that defined when `/` would be rendered as a literal slash
+versus treated as an operator.
+
+[a set of heuristics]: https://sass-lang.com/documentation/operators/numeric#slash-separated-values
+
+For a long time, these heuristics worked pretty well. In recent years, however,
+new additions to CSS such as [CSS Grid][] and [CSS Color Level 4][] have been
+using `/` as a separator increasingly often. Using the same character for both
+division and slash-separation is becoming more and more annoying to users, and
+will likely eventually become untenable.
+
+[CSS Grid]: https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row
+[CSS Color Level 4]: https://drafts.csswg.org/css-color/#rgb-functions
+
+## Summary
+
+> This section is non-normative.
+
+We will redefine `/` to be *only* a separator. Rather than creating an unquoted
+string (as it currently does when at least one operand isn't a number), it will
+create a slash-separated list. As such, lists will now have three possible
+separators: space, comma, and slash.
+
+Division will instead be written as a function, `divide()` (or `math.div()` in
```

Comment:
At what point is the top level `divide()` expected to be removed?

Reply:
In favor of `math.div()`? At the same time as other top-level functions, which is [very tentatively set to be 1 October 2021](https://github.com/sass/sass/blob/master/accepted/module-system.md#timeline).